battle-engine.js

```diff
@@ -495,7 +495,7 @@ BattlePokemon = (function () {
 		}
 		return null;
 	};
-	BattlePokemon.prototype.getMoveTargets = function (move, target) {
+	BattlePokemon.prototype.getMoveTargets = function (move, target, noRepeat) {
```

Comment:
idk, I don't want the function to accidentally infinite-loop into itself if someone configures a `ModifyMove` to change `pressureTarget` in a strange way and this is what I came up with because I'm uninspired.